src/app/components/splash-screen/splash-screen.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="app-splash-screen"
+[ngStyle] ="{'left': windowWidth}" >
```

Comment:
Avoid inline styles.

Reply:
Vaia, I used it to pass value from .ts to .scss dynamically. I have tries in several different way to pass value directly without via .html file, but it that case it becomes static, not dynamic.

Reply:
inline style removed vaia.